<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>1360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="passwordLine">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>530</y>
     <width>610</width>
     <height>98</height>
    </rect>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="accountLine">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>390</y>
     <width>610</width>
     <height>98</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="photoBtn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
  border-radius: 50px;
  background-size:cover;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="logBtn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>680</y>
     <width>471</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registBtn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>810</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>750</y>
     <width>531</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>已阅读并同意服务协议和QQ隐私保护指南</string>
   </property>
  </widget>
  <widget class="QPushButton" name="forgetBtn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>810</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>忘记密码</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>390</y>
     <width>211</width>
     <height>98</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
